eppo-server-sdk.gemspec

```diff
@@ -32,7 +32,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'rspec', '~> 3.12', '>= 3.12.0'
   s.add_development_dependency 'rubocop', '~> 1.41'
   s.add_development_dependency 'webmock', '~> 3.18', '>= 3.18.1'
-  s.required_ruby_version = '>= 3.1.2'
+  s.required_ruby_version = '>= 3.0.6'
```

Comment:
The reason this is 3.0.6 and not 3.0.0 is because that's the lowest official Dockerized Ruby version I could find: https://hub.docker.com/_/ruby

We could probably support on even lower versions if I use one of the unofficial images to test, but Carlos said this is enough to unblock Huckberry